package.json

```diff
@@ -38,25 +38,29 @@
     "lodash": "^4.16.1",
     "magic-string": "^0.19.0",
     "nsp": "^2.1.0",
+    "reify": "^0.8.3",
     "require-all": "^2.2.0",
     "rimraf": "^2.3.4",
     "rollup": "^0.41.1",
     "rollup-plugin-buble": "^0.15.0",
+    "rollup-plugin-commonjs": "^8.0.2",
     "rollup-plugin-filesize": "^1.0.0",
+    "rollup-plugin-multi-entry": "^2.0.1",
+    "rollup-plugin-node-resolve": "^3.0.0",
     "rollup-plugin-uglify": "^1.0.0",
     "tape": "^4.2.2",
     "watch": "^1.0.1"
   },
   "scripts": {
     "cov": "npm run cov:clean && npm run cov:generate",
     "cov:clean": "rimraf ./coverage/",
-    "cov:generate": "babel-node --presets=es2015 ./node_modules/.bin/istanbul cover test",
+    "cov:generate": "babel-node --presets=es2015 ./node_modules/.bin/istanbul cover ./dist/test.js",
     "prepublish": "npm run check",
     "pretest": "npm run build",
-    "test": "babel-node --presets=es2015 test && npm run lint",
+    "test": "node --require reify ./dist/test && npm run lint",
```

Comment:
The bundle is CJS but it doesn't matter since we don't export anything anyhow. But this would be the export of the bundle; it doesn't go thru and replace all the `import` statements in the tests.